src/commissaire/models/__init__.py

```diff
@@ -98,6 +98,18 @@ def new(cls, **kwargs):
         instance.__init__(**init_args)
         return instance
 
+    def __str__(self):  # pragma: no cover
+        """
+        Returns a string representation of the instance.
+
+        :returns: String representation of this instance.
+        :rtype: str
+        """
+        readable = '<{}'.format(self.__class__.__name__)
+        for k, v in self.to_dict().items():
```

Comment:
Not sure what this will be used for, but do you maybe want to use `to_dict_safe()` here?

IOW, should `'ssh_priv_key'` be excluded from a `Host` string representation?